src/diffusers/models/resnet.py

```diff
@@ -455,7 +455,6 @@ def __init__(
     ):
         super().__init__()
         self.pre_norm = pre_norm
```

Comment:
Actually we can remove this as well and the `pre_norm` in the init as well as it's never used